package.json

```diff
@@ -5,12 +5,10 @@
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start:rocket": "rocket start",
-    "build:rocket": "rocket build",
-    "start": "npm run clean && npm run start:rocket",
-    "build": "npm run clean && npm run build:rocket",
+    "start": "eleventy --serve",
+    "build": "npm run clean && eleventy",
     "bump": "npm --no-git-tag-version version",
-    "clean": "rm -rf _site"
+    "clean": "rm -rf dist"
```

Comment:
I know this was here previously, but would it make sense to use something like the **rimraf** for greater OS portability?  I think stock Windows specifically would fail to run this command, unless using something on top like WSL? (I could be wrong, and also maybe we don't anticipate any users to fit into this use case 🤷‍♂️ )